finish/src/main/java/io/openliberty/guides/beanvalidation/SerialNumber.java

```diff
@@ -20,16 +20,28 @@
 
 import javax.validation.Constraint;
 import javax.validation.Payload;
-
+// tag::Target[]
 @Target({ FIELD })
+// end::Target[]
+
+// tag::Retention[]
 @Retention(RUNTIME)
+// end::Retention[]
 @Documented
+// tag::Constraint[]
 @Constraint(validatedBy = { SerialNumberValidator.class })
-public @interface SerialNumber {
+// end::Constraint[]
 
+// tag::SerialNumber[]
+public @interface SerialNumber {
+    // tag::message[]
     String message() default "serial number is not valid.";
-
+    // end::message[]
+    // tag::groups[]
     Class<?>[] groups() default {};
-
+    // end::groups[]
+    // tag::payload[]
     Class<? extends Payload>[] payload() default {};
+    // end::payload[]
 }
+// end::SerialNumber[]
```

Comment:
add a new line

Reply:
Wouldn't the new line just be extra?

Reply:
It would be but the general practice is to keep the new line so that if someone wants to add the code the cursor by default is at new line.